tests/openvino/native/quantization/test_ptq_params.py

```diff
@@ -49,8 +49,8 @@ def get_ignored_patterns(device: TargetDevice = TargetDevice.ANY) -> GraphPatter
 # pylint: disable=protected-access
 @pytest.mark.parametrize("target_device", [TargetDevice.CPU, TargetDevice.GPU, TargetDevice.VPU])
 def test_target_device(target_device):
-    algo = PostTrainingQuantization(target_device=target_device)
-    min_max_algo = algo.algorithms[0]
+    pipelines = PostTrainingQuantization(target_device=target_device)
+    min_max_algo = pipelines.pipeline_steps[-1][0]
```

Comment:
This index hardcode doesn't explain clearly, how the `MinMax` algorithm should be get.
A good example of the increased pipeline complexity and the need for the `MinMax` as the independent unit.
cc @alexsu52